src/native/libs/System.Security.Cryptography.Native.Android/pal_jni.c

```diff
@@ -682,8 +682,7 @@ int GetEnumAsInt(JNIEnv *env, jobject enumObj)
     return value;
 }
 
-JNIEXPORT jint JNICALL
-JNI_OnLoad(JavaVM *vm, void *reserved)
+jint init_library_on_load (JavaVM *vm, void *reserved)
```

Comment:
I'd call this something else that includes the lib name so we don't end up with duplicate `init_library_on_load` if we add JNI_OnLoad to another library.

Something like `AndroidCryptoNative_InitLibraryOnLoad` would fit the existing naming convention better

Reply:
also mark it as `PALEXPORT`

Reply:
It doesn't have to be exported.  It's called by either android crypto's `JNI_OnLoad` when building a dynamic library or it will be called by our `Android for .NET'`s one if we link it in statically. In neither case it needs to be exported, it's an internal call.